addons/arsenal/XEH_postInit.sqf

```diff
@@ -1,2 +1,26 @@
 #include "script_component.hpp"
 
+GVAR(itemsPerModelHidden) = [createHashMap,createHashMap,createHashMap];
+GVAR(hiddenOptions) = createHashMap;
+
+{
+    private _classRoot = _x;
+    {
+        private _config = _x;
+
+        private _itemsPerModel = GVAR(itemsPerModelHidden) select (["CfgWeapons","CfgVehicles","CfgGlasses"] find _classRoot);
```

Comment:
To slightly improve performance you may move this line before the 2nd foreach scope, as it will be always the same result. Their you could use _foreachindex instead of find.

Reply:
Very true good find